SU-HSDO/stanford_mrc
# MRCD8-210 Updating with Decanter 2.0.0 stuff

# READY FOR REVIEW

# Summary
- Updated all of the necessary files and items in order to leverage new Decanter work
- Added stanford_basic to leverage Decanter that's imported on 8.x-2.x

# Needed By (Date)
- End of sprint (1/17)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Test making a change to a SCSS file and running grunt to verify that all if working well with the new Decanter code
3. Test on website (?)

# Affected Projects or Products
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-210
- Decanter
- stanford_basic

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
